### ISSUE TITLE
cuisine2 docker support not working well for ubuntu

### ISSUE THREAD

**Issue by despiegk** [Jumpscale/jumpscale_core8#86]
![image](https://cloud.githubusercontent.com/assets/6021844/13545959/233c086c-e2b7-11e5-902d-1a9f99accb31.png)

not implemented , see todo
I know things have been implemented where are they

+ I needed to do quite some workarounds to get this going

test script for all of this
```
from JumpScale import j

pushkey=False

if pushkey:
    #push local key to remove host, use passwd to login first time
    c=j.tools.cuisine.getPushKey(addr='someserver:2222', login='root', passwd='something', keyname='mykey') 
else:

    c=j.tools.cuisine.get('alinuxhost:2222') #modified for security


print ("install jumpscale, this will take a while")
#be carefull this will remove history, will be installed in read write mode
c.installer.jumpscale8(rw=True, reset=True)


print ("INSTALL DOCKER, THIS WILL TAKE A WHILE")
#install docker
c.docker.install()

#build ubuntu from docker file
print ("BUILD UBUNTU WITH SSH SUPPORT")
c.docker.ubuntuBuild()
pubkey=j.do.getSSHKeyFromAgentPub("ovh_install")

print ("LAUNCH UBUNTU DOCKER WITH MY PUB LOCAL SSH KEY")
c2=c.docker.ubuntu(name="ubuntu_test1", image='jumpscale/ubuntu1510', ports=None, volumes=None, pubkey=pubkey, aydofs=False)
#c2 is cuisine client then to the installed ubuntu docker






from IPython import embed
print ("DEBUG NOW id")
embed()

```

**Comment by despiegk** [comment]
i've now enough workarounds to continue, but the issue is still there, the jsdocker does not work well and we need support for js8

**Comment by despiegk** [comment]
@zaibon ?

**Comment by zaibon** [comment]
should be fix by #102. need review
